opentracing/scope.py

```diff
@@ -18,7 +18,7 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-from __future__ import absolute_import
```

Comment:
Do we really need to remove this one?

Reply:
It just takes no effect here, so I considered it redundant.
Do you want me to restore this line?

Reply:
let's not change things that don't relate to this PR

Reply:
This line has been restored.